packages/tests/src/web-test-runner.config.ts

```diff
@@ -169,7 +169,9 @@ const require = createRequire(import.meta.url);
 
 // https://modern-web.dev/docs/test-runner/cli-and-configuration/
 const config: TestRunnerConfig = {
-  rootDir: '../',
+  // Serve from the root of the monorepo, because most dependencies are hoisted
+  // into its node_modules folder.
+  rootDir: '../../',
```

Comment:
I thought WTR could serve files outside of the root with a special endpoint?

Reply:
It does, but for our HTML tests we use hard-coded paths to node_modules. Added a comment.